docs/self-audit/scripts/self-audit.sh

```
@@ -0,0 +1,461 @@
+    local phase_result="$OUTPUT_DIR/results/phase0-dependencies.json"
+    
+    if [ "$VERBOSE" = true ]; then
+        log_info "npm audit実行中..."
+        log_info "ライセンス監査実行中..."
+        log_info "パッケージバージョン監査実行中..."
+    fi
+    
+    node scripts/audit-phase0-dependencies.js \
+        --output "$phase_result" \
+        --format "$OUTPUT_FORMAT" \
+        $([ "$VERBOSE" = true ] && echo "--verbose") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)